src/api/services/media_library.py

```diff
@@ -68,13 +68,13 @@ def getImageInfo(self, args):
         events = serialize_all(media.events.all())
         actions = serialize_all(media.actions.all())
         testimonials = serialize_all(media.testimonials.all())
-        media_json = get_json_if_not_none(media)
+        media_json = get_json_if_not_none(media, True)
         return {
             **media_json,
-            "user_info": user_info,
-            "info": {
-                "events": events,
-                "actions": actions,
-                "testimonials": testimonials,
+            "information": user_info,
```

Comment:
i think user_info is more explicit.  information is very generic.  You can also go for `user_information`